Makefile

```diff
@@ -1,8 +1,5 @@
 PYTHON ?= python
```

Comment:
I don't like Makefiles but if we need them for another release or two then I like this change. +1